### PR TITLE
#164 add_display_name #164

### PR DESCRIPTION
**Pull request by ank1traj**
**PR Title: Add display_name field to student and mentor models**

**PR Description:**

This PR adds a `display_name` field to student and mentor models. This field will be displayed on the website and in statistics while the `name` field will be printed on the certificate.

**Changes:**

* Add a `display_name` field to the `Student` and `Mentor` models.

**Benefits:**

* This change will make it easier for users to identify students and mentors on the website and in statistics.
* It will also make it easier to distinguish between the `display_name` and `name` fields when printing certificates.

### CODE REVIEW

File: .githooks/pre-commit
Comment (harshkhandeparkar) [inline .githooks/pre-commit:1]: Why was this file changed?

File: README.md [hunk 1]
```diff
@@ -263,6 +263,7 @@ The `models/` directory contains database models for the KWoC database tables. (
   - `name` (string): Name of the mentor.
   - `email` (string): Email of the mentor.
   - `username` (string): Username of the mentor.
+  - `display_name` (string): display name of the mentor
```
Comment (harshkhandeparkar) [inline README.md:266]:
```suggestion
  - `display_name` (string): The display name of the mentor.
```

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

**Comment by harshkhandeparkar** [comment]
Have you tested this locally?

**Comment by harshkhandeparkar** [comment]
@ank1traj are you still working on this? If yes, please make the requested changes. Thanks!

**Comment by harshkhandeparkar** [comment]
@ank1traj ?

File: .githooks/pre-commit
Comment (ank1traj) [reply .githooks/pre-commit:1]: Din't make any changes in this file. It got deleted so just added with same code.

**Comment by ank1traj** [comment]
> Have you tested this locally?

Couldn't able to check due to the laptop constraints. And, sorry for missing out the above message. Didn't get the notifications before.

File: .githooks/pre-commit
Comment (harshkhandeparkar) [reply .githooks/pre-commit:1]: File permission seems to have changed. Can you revert this change?

**Comment by harshkhandeparkar** [comment]
@ank1traj The tests were recently fixed in #179. Can you please rebase on the main branch? Thanks.

**Comment by harshkhandeparkar** [comment]
@ank1traj ?

**Pull request closed without merging**